python/src/aiconfig/Config.py

```diff
@@ -53,11 +52,14 @@
 ModelParserRegistry.register_model_parser(
     DefaultAnyscaleEndpointParser("AnyscaleEndpoint")
 )
+ModelParserRegistry.register_model_parser(
+    # TODO: Get a list of default models for AzureOpenAIParser
+    AzureOpenAIParser(deployment="gpt-35-turbo")
+)
 ModelParserRegistry.register_model_parser(
     GeminiModelParser("gemini-pro"), ["gemini-pro"]
 )
 ModelParserRegistry.register_model_parser(ClaudeBedrockModelParser())
-ModelParserRegistry.register_model_parser(HuggingFaceTextGenerationParser())
```

Comment:
I think this will break the current HuggingFace/Mistral-aiconfig.json cookbook. Can you check that? Looks like it has an hf.py there but defines the parser as HuggingFaceTextParser -- probably just need to update the config there to use 'HuggingFaceTextParser' instead.

Reply:
Yea good point, I'll just not remove the main HF parser from this PR and do that in another one in the future. Issue: https://github.com/lastmile-ai/aiconfig/issues/1116